include/dfu/mcuboot.h

```diff
@@ -65,16 +65,14 @@ extern "C" {
 #define BOOT_IMG_VER_STRLEN_MAX 25  /* 255.255.65535.4294967295\0 */
 
 /* Trailer: */
-#ifndef CONFIG_FLASH_ALIGNMENT_WORD_SIZE_BYTES
-#warning "expecting flash alignment to be defined, using 8 bytes as default"
-#define CONFIG_FLASH_ALIGNMENT_WORD_SIZE_BYTES 8
-#endif
+#define FLASH_WRITE_BLOCK_SIZE \
```

Comment:
I think the zephyr unittests fail because there is no default FLASH_WRITE_BLOCK_SIZE set to 8 when there is no DT node.

Reply:
@JvanDooren Unit tests fixed. It required a small tweak to MCUBoot.